src/SCRIPTS/BF/ui.lua

```diff
@@ -358,6 +358,7 @@ local function run_ui(event)
             end
         elseif pageState == pageStatus.editing then
             if event == EVT_VIRTUAL_EXIT or event == EVT_VIRTUAL_ENTER then
+                pcall(Page.fields[currentField].postEdit, Page)
```

Comment:
Why do we need the `pcall()` here?

Reply:
It just prevents the script from crashing if the field doesn't have ```postEdit```. 
Could have done this too
```
if Page.fields[currentField].postEdit then
    Page.fields[currentField].postEdit(Page)
end
```

Reply:
I actually think this is a better way, as it is more specific. Otherwise any error that snuck into a `postEdit()` function could go undetected for a long time, causing debug nightmares.

Reply:
Good point! I'll change it